cam0:
  cam_overlaps: [1]
  camera_model: pinhole
  distortion_coeffs: [-0.17502173882417144, 0.028388033197885253, -0.00033687391709979743,
    -0.000589715880254496]
  distortion_model: radtan
  intrinsics: [699.2106252004894, 698.0949264742881, 648.1876657209569, 332.07811883673014]
  resolution: [1280, 720]
  rostopic: /camera/left/image_raw
cam1:
  T_cn_cnm1:
  - [0.9999425812141998, 4.9322002503504545e-05, 0.010715962020434212, -0.11955398554901213]
  - [0.00016974748755216497, 0.9997910380173701, -0.020441415947528276, -0.0004510752486116766]
  - [-0.010714731003333176, 0.02044206123387421, 0.9997336228576273, 0.001019669395571752]
  - [0.0, 0.0, 0.0, 1.0]
  cam_overlaps: [0]
  camera_model: pinhole
  distortion_coeffs: [-0.16202558421329066, 0.014204420753601139, -0.0007071824198034041,
    -0.0010462229878008876]
  distortion_model: radtan
  intrinsics: [697.4169797695323, 697.2822316251281, 644.9573960144035, 351.31362096635223]
  resolution: [1280, 720]
  rostopic: /camera/right/image_raw
